project.clj

```diff
@@ -1,4 +1,4 @@
-(defproject ovotech/kafka-avro-confluent "1.1.1-3"
+(defproject ovotech/kafka-avro-confluent "1.2.0"
```

Comment:
we're keeping these in sync with kafka versions since otherwise it's a complete pain in the ass to manage. so this would be `1.1.1-4`, which isn't ideal as it just communicates a re-build .. for that reason, idk if it's worth publishing branch that uses `ovotech/abracad` and one that uses `com.damballa/abracad`? or just do bump it to `1.1.1-4` and document the breaking change in the top of the readme in a bolder way ?

Reply:
My bad for not reading the docs properly. It seems like I/we might take over ownership of the repo anyway so it might be best to just bump it to 1.1.1-4